data-jdbc/src/test/groovy/io/micronaut/data/jdbc/oraclexe/jsonview/OracleXEJsonViewSpec.groovy

```diff
@@ -0,0 +1,293 @@
+        studentScheduleClassView.setRoom(classMath.getRoom())
+        studentScheduleClassView.setTeacher(teacherView)
+
+        newStudentScheduleView.setClazz(studentScheduleClassView)
+        ivoneStudentView.setSchedule(List.of(newStudentScheduleView))
+        peterStudentView.setSchedule(List.of(newStudentScheduleView))
+        studentViewRepository.save(ivoneStudentView)
+        studentViewRepository.saveAll(Arrays.asList(peterStudentView))
+
+        def optIvoneStudentView = studentViewRepository.findByName(ivoneStudentName)
+        def optPeterStudentView = studentViewRepository.findById(peterStudentView.id)
+        def clazz = classRepository.findByName(className)
+
+        then:
+        optPeterStudentView.present
+        optIvoneStudentView.isPresent()
+        // And just to validate that saved local time is + 30 minutes from initial class time
+        def studentClassTime = optIvoneStudentView.get().getSchedule().get(0).getClazz().getTime()
+        classTime.plusMinutes(30) == studentClassTime
+        // And also in class table itself
+        def updatedClassTime = clazz.getTime()
+        classTime.plusMinutes(30) == updatedClassTime
+
+        when:
+        studentViewRepository.deleteAll()
+        def count = studentViewRepository.count()
+        then:
+        count == 0
+    }
+
+    def "delete record"() {
+        when:
+        def studentName = "Denis"
+        def optionalStudentView = studentViewRepository.findByName(studentName)
+        then:
+        optionalStudentView.present
+
+        when:
+        studentViewRepository.deleteById(optionalStudentView.get().id)
+        optionalStudentView = studentViewRepository.findByName(studentName)
+        then:
+        !optionalStudentView.present
+
+        when:"Verify via regular repo"
+        def optionalStudent = studentRepository.findByName(studentName)
+        then:
+        !optionalStudent.present
+
+        when:
+        optionalStudentView = studentViewRepository.findByName("Josh")
+        def count = studentViewRepository.count()
+        then:
+        optionalStudentView.present
+        count > 0
+        when:
+        studentViewRepository.deleteAll(Arrays.asList(optionalStudentView.get()))
+        optionalStudentView = studentViewRepository.findByName("Josh")
+        then:
+        !optionalStudentView.present
+
+        when:
+        def optFredStudentView = studentViewRepository.findByName("Fred")
+        studentViewRepository.delete(optFredStudentView.get())
+        optFredStudentView = studentViewRepository.findByName("Fred")
+        count = studentViewRepository.count()
+        then:
+        // After deleted should not be present
+        !optFredStudentView.present
+        count == 0
+    }
+
+    static OracleContainer createContainer() {
+        return new OracleContainer(DockerImageName.parse("gvenzl/oracle-free:slim-faststart").asCompatibleSubstituteFor("gvenzl/oracle-xe"))
```

Comment:
I have tried, it's not starting up with test-containers for some reason. Tried some other as well, but also not starting up. 
All images are pretty large and take time to start up, not sure tests will run on github successfully. Until test containers have support for 23c there won't be nice solution I think.

Reply:
Hi @radovanradic,

There was a bug in the images released last Saturday which has been fixed this early morning. The images should work as expected again.

As to whether these images run successfully on GitHub, there are certainly other frameworks that use these images already successfully (although I can't comment on whether inside or outside Testcontainers) on GitHub via GitHub Actions.

I can help with the Micronaut test suite if you run into any issues.

Reply:
Thanks @gvenzl . I pulled again some of the images and they are starting up now.
The issue I am having is probably related to the image compatibility in test containers.

Reply:
Not sure exactly what is going on with this workaround 
```
new OracleContainer(DockerImageName.parse("gvenzl/oracle-free:latest-faststart").asCompatibleSubstituteFor("gvenzl/oracle-xe"))
                .withDatabaseName("test").withInitScript("./oracle-json-view-init.sql")
```
as this is needed to run test container with this new image. Not sure if it is taking too long to pull and start container, I have only one occurrence of passing test in github. We also have specific situation with test resources/containers where oracle-xe image is starting here even if we don't need it and I didn't find the way to avoid it. @dstepanov do you know if we can have test in data-jdbc that will skip test resources somehow?